code/modules/assembly/health.dm

```diff
@@ -49,22 +49,23 @@
 	if(!scanning || !secured)
 		return
 
-	var/atom/A = src
+	var/atom/object = src
 	if(connected?.holder)
-		A = connected.holder
-	for(A, A && !ismob(A), A=A.loc);
+		object = connected.holder
+	for(object, object && !ismob(object), object=object.loc);
```

Comment:
god I hate the syntax
do me a favor could you replace this with a while loop, people are way more likely to actually understand that

Reply:
Ahhh